heathskin/entity.py

```diff
@@ -49,29 +49,23 @@ def update_tag(self, tag_name, tag_value):
         else:
             old_str = "None"
 
-         # using integer sequence for ordering until timestamp available from parser
-        event_sequence = len(self._tag_history)
-        #                   def __init__(self, index, tag_name, old_value, new_value):
         new_event = EntityChangeEvent(tag_name, old_str, tag_value)
-        self._tag_history.append(new_event)
+        self.tag_history.append(new_event)
 
         self.logger.info(
-            "Updating %s:%s [%s->%s]",
-            self.__repr__(), tag_name, old_str, tag_value)
+            "Updating %s %s",
+            self.__repr__(), new_event.__str__())
```

Comment:
Didn't change that because there's no __str__() method on entity, and I figured whoever added __repr__() wanted to keep it that way.

did new_event.__str__() to match the prior argument.